src/lib.rs

```diff
@@ -484,7 +563,48 @@ impl Printer for PrintlnPrinter {
 
 struct NullLoader;
 impl ModuleScriptLoader for NullLoader {
-    fn load_script(&mut self, _: String) -> Option<String> { None }
+    fn load_script(&mut self, _: String) -> Option<String> {
+        None
+    }
+}
+
```

Comment:
Make the code below the only change, or better yet, offload both `ModuleScriptLoader`s to a separate module, and `pub use` them at the top. I plan to eventually reduce the monolith that is currently `lib.rs` anyways.